src/main/java/io/rcktapp/api/service/ext/RedisHandler.java

```diff
@@ -0,0 +1,171 @@
+package io.rcktapp.api.service.ext;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.forty11.j.J;
+import io.forty11.web.js.JS;
+import io.forty11.web.js.JSObject;
+import io.rcktapp.api.Action;
+import io.rcktapp.api.Api;
+import io.rcktapp.api.Chain;
+import io.rcktapp.api.Endpoint;
+import io.rcktapp.api.Handler;
+import io.rcktapp.api.Request;
+import io.rcktapp.api.Response;
+import io.rcktapp.api.SC;
+import io.rcktapp.api.service.Service;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+
+/**
+ * @author kfrankic
+ *
+ */
+public class RedisHandler implements Handler
+{
+   Logger                log               = LoggerFactory.getLogger(RedisHandler.class);
+
+   private String        host              = "";
+   private int           port              = 0;
+
+   // times in milliseconds
+   private int           readSocketTimeout = 0;
+
+   final JedisPoolConfig poolConfig        = buildPoolConfig();
+   JedisPool             jedisPool         = null;
+
+   @Override
+   public void service(Service service, Api api, Endpoint endpoint, Action action, Chain chain, Request req, Response res) throws Exception
+   {
+      if (jedisPool == null)
+         jedisPool = new JedisPool(poolConfig, host, port, readSocketTimeout);
+
+      Jedis jedis = jedisPool.getResource();
+      
+      // the key is derived from the URL
+      String url = removeUrlProtocol(req.getApiUrl()) + req.getPath() + sortRequestParameters(req.getParams());
+      String key = J.md5(url.getBytes());
```

Comment:
Let's not MD5 the key

```suggestion
      String key = url.getBytes();
```